src/extension.ts

```diff
@@ -151,6 +151,7 @@ class AmigaDebugExtension {
 			vscode.commands.registerCommand('amiga.externalResources.colorReducer', () => MinimalBrowser.launchUrl('http://deadliners.net/ColorReducer', 'Color Reducer')),
 			vscode.commands.registerCommand('amiga.externalResources.bltconCheatSheet', () => MinimalBrowser.launchUrl('http://deadliners.net/BLTCONCheatSheet', 'BLTCON Cheat Sheet')),
 			vscode.commands.registerCommand('amiga.externalResources.amigaHRM', () => MinimalBrowser.launchUrl('http://amigadev.elowar.com/read/ADCD_2.1/Hardware_Manual_guide/node0000.html', 'Amiga Hardware Reference Manual')),
+			vscode.commands.registerCommand('amiga.programFolder', () => path.dirname(vscode.workspace.getConfiguration('amiga').get('program')?.toString().replace('\\', '/'))),
```

Comment:
I would suggest to use `.replace(path.sep, '/')`. I know it's rare, but since this runs on Unixes now, replacing `path.sep` with forward slashes seems safer (also communicates intent better)

Reply:
You're right. Thank you for the review :) And sorry for the late answer.
ATM this pull request is far away from head, and it looks easier to rewrite than to merge.
Additionally, there are changes related to Linux that I don't know exactly how to re-do (I am not a Linux user, if I run a new pull-request the part regarding Shrinkler will be working on Windows only). May you give me some advice on it? :)

Reply:
Please find an up-to-date version of the PR upgraded to 1.6.8, including the change you've requested.
https://github.com/BartmanAbyss/vscode-amiga-debug/pull/200